solc/CommandLineInterface.cpp

```diff
@@ -158,7 +158,10 @@ static bool needsHumanTargetedStdout(CommandLineOptions const& _options)
 		_options.compiler.outputs.natspecDev ||
 		_options.compiler.outputs.opcodes ||
 		_options.compiler.outputs.signatureHashes ||
-		_options.compiler.outputs.storageLayout;
+		_options.compiler.outputs.storageLayout ||
+		_options.compiler.outputs.ir ||
+		_options.compiler.outputs.irOptimized ||
```

Comment:
Ah, see, this is gonna be a problem, i.e. this approach is not gonna work. We have some two-stage tests where the input file is compiled to yul, and then compiled again; during the first compilation, a temporary file will be created where the ```====file====``` text will be added on top, and then when we try to compile it for the second time around, the compilation obviously fails because this isn't a valid statement.
I'll have to dig around and see whether it's even possible for us to introduce this behaviour without major changes in the way we test.

Reply:
I have removed the changes to the header output and the associated cmdline tests. This is now just the output when there is no contract provided issue.